docs/source/networks.rst

```diff
@@ -0,0 +1,311 @@
+Many experiments will be able to just use one of Dallinger's existing
+networks, rather than defining their own. Lets look at the basic networks
+that can be used out of the box.
+
+Empty
+^^^^^
+
+There are experiments where participants do not need to interact with each
+other at all. Generally, in this case, a source will be required. The empty
+network does not connect any nodes with each other, which results in a
+series of isolated nodes. The only exception is, if a source node is added,
+it will be connected to all existing nodes, which means that it's possible to
+send a stimulus to all network nodes, regardless of their isolation.
+
+Chain
+^^^^^
+
+A chain network, also known as `line` network, connects each new node to the
+previous one, so that nodes can receive information from their parent, but
+cannot send information back. In other words, it's a one way transmission
+chain. In general, it's useful to have a source as the first node, so that
+an initial experiment stimulus is transmitted to the each node through the
+chain. Note that this network explicitly prohibits a source to be added after
+any node, so the source has to come first.
+
+This network can be useful for experiments where some piece of information,
+for example, a text, needs to be modified or interpreted by each participant
+in succession.
+
+Star
+^^^^
+
+A start network uses its first node as a central node, and nodes created
+after that have a bidirectional connection (`both`) with that node. This
+means the central node can send and receive information from/to all nodes,
+but every other node in the network can only communicate with the central
+node.
+
+A source can't be used as a first node, since the connections to it need to
+be in both directions.
+
+Burst
+^^^^^
+
+A burst network is very similar to a star network, except the central node is
+connected to the other nodes using a `to` connection. In this case, a source
+can be used as a central node.
+
+FullyConnected
+^^^^^^^^^^^^^^
+
+A fully connected network is one where all the nodes are connected to each
+other in both directions, thus allowing any node to transmit and receive from
+any other node. This can be very useful for cooperation experiments or
+chatrooms.
+
+A source is allowed as a node in this network, and it will use a `to`
+connection to the other nodes, so transmitting to it will not be allowed.
+
+Other available networks
+------------------------
+
+There are other, somewhat more specialized networks that an experiment can
+use. Here's a quick rundown.
+
+DelayedChain
+^^^^^^^^^^^^
+
+The delayed chain acts as a sort of burst network for the first eleven nodes,
+but turns into a chain network for all nodes after that. If the network has a
+source, it is assigned as a parent to the first eleven nodes. From the
+twelfth node onwards, the previous node is used as a parent, using a `to`
+connection, like in the chain network. If there's no source, the first eleven
+nodes have no parent.
```

Comment:
I don't think we should compare this to a burst network. It was created as some MTurk configurations require you to recruit at least 10 people at the start. If you want a Chain network but you have 10 people all at once you'd have to make them wait their turn. If the experiment takes 5 minutes, the last person will be waiting for almost an hour.

This is intended to be used with a Source, in order to form a long running chain where users are recruited as and when the previous person finishes. This could potentially be hundreds of users long and take place over many days, as it doesn't require synchronisation, but in order to do that recruitment you need to get around the initial 10 user problem.

I'd describe this as a chain that ignores the first 9 nodes that are added, to suit MTurk's large recruitment settings.

Maybe a diagram would help?

             +--+
          +> |02|
          |  +--+
          +> |03|
          |  +--+
          +> |04|
          |  +--+
          +> |05|
          |  +--+
          +> |06|
    +--+  |  +--+
    |01+---> |07|
    +--+  |  +--+
          +> |08|
          |  +--+
          +> |09|
          |  +--+
          +> |10|
          |  +--+    +--+    +--+    +--+
          +> |11+--->+12+--->+13+--->+14| ...
             +--+    +--+    +--+    +--+